OutOfSchool/OutOfSchool.WebApi/Extensions/MappingExtensions.cs

```diff
@@ -166,6 +167,11 @@ public static WorkshopDTO ToModelSimple(this Workshop workshop)
             });
         }
 
+        public static ElasticsearchSyncRecordDto ToModel(this ElasticsearchSyncRecord elasticsearchSyncRecord)
```

Comment:
Approach with constructing Mapper on-the-fly is deprecated (due to perf. reason). Please inject mapper dependency and configure AutoMapper on Startup

Reply:
fixed